common/decklist.cpp

```diff
@@ -466,66 +466,111 @@ bool DeckList::saveToFile_Native(QIODevice *device)
 bool DeckList::loadFromStream_Plain(QTextStream &in)
```

Comment:
This function is massive, it would be good to break it up into smaller units that are more easily composed and tested in isolation